modules/local/fasta/main.nf

```diff
@@ -8,8 +8,8 @@ process FASTA {
         'quay.io/biocontainers/bedtools:2.30.0--h7d7f7ad_2' }"
 
     input:
-    tuple val(meta), path(bed)
-    path fasta
+    tuple val(meta), path(bed, name: "bed.input")
+    path fasta, name: "fasta.input"
 
```

Comment:
why are you naming the input here?

Reply:
I think this should better be handled using `task.ext.prefix` in the `modules.config` - will quickly do that

Reply:
Ok there was not even a file name collission, don't know why I did that

Reply:
Okay I found the reason for this now: There are parts of this process which manipulate all files that end with *.fa and *.fasta. If the input has one of this extensions, it will not work properly. I temporarily re-activated this, since restructuring of the process is not in the scope of this PR but I will handle this soon.